src/init.cpp

```diff
@@ -71,13 +71,15 @@
 #include <zmq/zmqrpc.h>
 #endif
 
+namespace {
 bool fFeeEstimatesInitialized = false;
-static const bool DEFAULT_PROXYRANDOMIZE = true;
-static const bool DEFAULT_REST_ENABLE = false;
-static const bool DEFAULT_STOPAFTERBLOCKIMPORT = false;
+const bool DEFAULT_PROXYRANDOMIZE = true;
+const bool DEFAULT_REST_ENABLE = false;
+const bool DEFAULT_STOPAFTERBLOCKIMPORT = false;
 
 // Dump addresses to banlist.dat every 15 minutes (900s)
-static constexpr int DUMP_BANS_INTERVAL = 60 * 15;
+constexpr int DUMP_BANS_INTERVAL = 60 * 15;
+} // namespace
```

Comment:
Why is this changed? What is the difference between `static const(expr) int` and `const int` in a namespace?

We don't change code for stylistic reasons.

Reply:
Also, I cant see a reason to not have literals in the global namespace

Reply:
static inside a anonymous namespace is redundant.

Reply:
Please see also https://github.com/bitcoin/bitcoin/pull/15622#discussion_r267802687

Let me know if you find any reason to revert to the original version.

Reply:
> Also, I cant see a reason to not have literals in the global namespace

If we want to share say `DUMP_BANS_INTERVAL` to another compilation unit the recommended way to do it is via `#include "init.h"`, no?

Given the amount of hate we get due to the said environmental impact of SHA256² we don't want to make things worse by polluting also the global namespace! :-)

Reply:
>  static inside a anonymous namespace is redundant.

ftfy: anonymous namespace is redundant when the constexpr is already static